config/scout_apm.yml

```diff
@@ -19,19 +19,20 @@ common: &defaults
   # - Default: none
   # - Valid Options: true, false
   monitor: true
+  auto_instruments: true
 
 production:
-  auto_instruments: true
   <<: *defaults
 
 development:
   <<: *defaults
   monitor: false
+  auto_instruments: false
```

Comment:
@Mahaveer1141 - I believe if `monitor` if `false`, then we don't even need to specify `auto_instruments`.

Reply:
ok, updated this